x/distribution/keeper/grpc_query.go

```diff
@@ -354,11 +354,11 @@ func (k Querier) DelegatorWithdrawAddress(ctx context.Context, req *types.QueryD
 }
 
 // CommunityPool queries the community pool coins
-func (k Querier) CommunityPool(ctx context.Context, req *types.QueryCommunityPoolRequest) (*types.QueryCommunityPoolResponse, error) {
+func (k Querier) CommunityPool(ctx context.Context, req *types.QueryCommunityPoolRequest) (*types.QueryCommunityPoolResponse, error) { //nolint:staticcheck // we're using a deprecated call for compatibility
 	pool, err := k.FeePool.Get(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	return &types.QueryCommunityPoolResponse{Pool: pool.CommunityPool}, nil
+	return &types.QueryCommunityPoolResponse{Pool: pool.CommunityPool}, nil //nolint:staticcheck // we're using a deprecated call for compatibility
```

Comment:
this will return empty right? should we pass a query router to distr to be able to query the pool module?